README.md

```diff
@@ -325,8 +354,6 @@ The formatted code will be returned on stdout (unless `--check` was
 passed).  *Black* will still emit messages on stderr but that shouldn't
 affect your use case.
 
-This can be used for example with PyCharm's [File Watchers](https://www.jetbrains.com/help/pycharm/file-watchers.html).
```

Comment:
Don't remove this. File Watchers is a distinct and also useful integration if somebody wants to format on save.